### ISSUE TITLE
Django 2.2 compatibility

### ISSUE THREAD

**Issue by michaelchen21** [PragmaticMates/django-flatpages-i18n#21]
I just installed django-flatpages-i18n for a Django 2.2 project. I followed the documentation through the "Migrate your database step", but after running `python manage.py migrate`, I got the following traceback:

```
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 381, in execu
te_from_command_line
    utility.execute()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 357, in execu
te
    django.setup()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_u
nlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_mo
dule
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frame
s_removed
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\models.py", line 11, in <module>
    class FlatPage_i18n(MPTTModel):
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\models.py", line 15, in FlatPage_i18n
    null=True, blank=True)
TypeError: __init__() missing 1 required positional argument: 'on_del
ete'
```

Adding a bunch of `, on_delete=models.CASCADE` to the `ForeignKey`s in `flatpages_i18n\models.py` resolved this issue.

**Comment by michaelchen21** [comment]
Adding a bunch of `, on_delete=models.CASCADE` to the `ForeignKey`s in `flatpages_i18n\models.py` resolved the issue.

**Comment by michaelchen21** [comment]
Then I got:

```
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 381, in execu
te_from_command_line
    utility.execute()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 357, in execu
te
    django.setup()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\apps\registry.py", line 114, in populate
    app_config.import_models()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\apps\config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_u
nlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_mo
dule
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frame
s_removed
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\models.py", line 51, in <module>
    class MenuItem(MPTTModel):
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\models.py", line 63, in MenuItem
    null=True, blank=True)
TypeError: __init__() missing 1 required positional argument: 'on_del
ete'

C:\Users\micha\Documents\Git\flushingeyes>python manage.py migrate
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 381, in execu
te_from_command_line
    utility.execute()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\core\management\__init__.py", line 357, in execu
te
    django.setup()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\apps\registry.py", line 122, in populate
    app_config.ready()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\contrib\admin\apps.py", line 24, in ready
    self.module.autodiscover()
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\contrib\admin\__init__.py", line 26, in autodisc
over
    autodiscover_modules('admin', register_to=site)
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\django\utils\module_loading.py", line 47, in autodiscov
er_modules
    import_module('%s.%s' % (app_config.name, module_to_search))
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_u
nlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_mo
dule
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frame
s_removed
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\admin.py", line 12, in <module>
    from flatpages_i18n.widgets import RedactorEditor
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\widgets.py", line 5, in <module>
    from django.core.urlresolvers import reverse
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```

I changed a line in `widgets.py` from `from django.core.urlresolvers import reverse` to `from django.urls import reverse`.

Then I got:

```
...
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\templatetags\flatpages_i18n.py", line 16
0, in <module>
    @register.assignment_tag(takes_context=True)
AttributeError: 'Library' object has no attribute 'assignment_tag'
```

Following the advice in [this Stack Overflow question](https://stackoverflow.com/questions/21021690/python-django-custom-template-tags-register-assignment-tag-not-working), I changed that to `simple_tag`.

```
  File "C:\Users\micha\AppData\Local\Programs\Python\Python37-32\lib\
site-packages\flatpages_i18n\migrations\0001_initial.py", line 49, in
 Migration
    ('parent', mptt.fields.TreeForeignKey(related_name='children', bl
ank=True, to='flatpages_i18n.FlatPage_i18n', null=True)),
TypeError: __init__() missing 1 required positional argument: 'on_delete'
```

Added another `on_delete=models.CASCADE`.

**Comment by michaelchen21** [comment]
Hmm, I was going to make a pull request, but it seems that the version from GitHub and the version from Pip are subtly different in many lines.

**Comment by eriktelepovsky** [comment]
Hi. It was already fixed in the pull request #20
